### ISSUE TITLE
Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by yss14** [sunknudsen/npm-sync#1]
Hi! It's me again :D

Just wanted to try out `npm-sync`, but I receive the following error:

```
npm i -g npm-sync
npm-sync --dest /Users/yss/Dev/opensource/<my-project> --watch --no-deps --yes --verbose
Cannot convert undefined or null to object
```

System: macOS 11.1
Node: 15.6.0
rsync: 2.6.9 (preinstalled by macOS)

**Comment by sunknudsen** [comment]
Hey,

Thanks for the bug report and for contributing to the development of `npm-sync`. 🤓

I improved error reporting when `--verbose` is set. Can you please update to `2.0.0-beta.4` and share the stack trace here?

Thanks,

Sun

**Comment by yss14** [comment]
```
/Users/yss/.nvm/versions/node/v15.6.0/lib/node_modules/npm-sync/build/index.js:52
        const packagePeerDeps = Object.keys(packageProperties.peerDependencies);
                                       ^

TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at sync (/Users/yss/.nvm/versions/node/v15.6.0/lib/node_modules/npm-sync/build/index.js:52:40)
```

Seems like there should be a fallback value for `peerDependencies`?

**Comment by sunknudsen** [comment]
> Seems like there should be a fallback value for `peerDependencies`?

I totally agree, on it!

**Comment by sunknudsen** [comment]
Fixed in `2.0.0-beta.5`
